nifi-nar-bundles/nifi-salesforce-bundle/nifi-salesforce-processors/src/main/java/org/apache/nifi/processors/salesforce/PutSalesforceObject.java

```diff
@@ -170,24 +177,30 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                     out.reset();
                 }
             }
-
             if (writer.isActiveRecordSet()) {
                 processRecords(objectType, out, writer, extender);
             }
-            session.transfer(flowFile, REL_SUCCESS);
-
+          }
+          session.transfer(flowFile, REL_SUCCESS);
+          final long transferMillis = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startNanos);
```

Comment:
The whole bundle does not use the final keyword. Please remove it for consistency.